src/components/Panel/Panel.tsx

```diff
@@ -37,7 +52,18 @@ const Wrapper = styled.div<{
   $hasShadow?: boolean;
   $color?: PanelColor;
   $padding?: PanelPadding;
+  $width?: string;
+  $fillWidth?: boolean;
+  $height?: string;
+  $fillHeight?: boolean;
+  $orientation?: Orientation;
 }>`
+  display: flex;
+  flex-flow: ${({ $orientation = "horizontal" }) =>
+    $orientation === "horizontal" ? "row wrap" : "column"};
+  align-items: flex-start;
```

Comment:
For horizontal its good to have align-items: center

Reply:
in the current design we align items at the top see here

<img width="788" alt="image" src="https://github.com/ClickHouse/click-ui/assets/1456541/a335637a-64e8-4687-b6a4-a3a8abf2efb9">

Reply:
We should then have a align prop
Since the places I'm using panel in sql-console needs it 
Like the clickpipes UI

Reply:
added